examples/apps/odspsnapshotfetch-perftestapp/package.json

```diff
@@ -23,9 +23,12 @@
     "dev": "npm run webpack:dev",
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+    "format": "npm run prettier:fix",
     "lint": "npm run eslint",
```

Comment:
Should we update the lint checks here too? Or are you doing that separately?

Reply:
I think it is a good practice to stick with the format enforced in other directories, changed to include `prettier`, and `prettier:fix` in `lint` and `lint:fix` respectively.